tests/plugins/emoji/basic.js

```diff
@@ -210,7 +225,19 @@
 				} );
 
 				arrayAssert.itemsAreSame( expected, actual );
-				assert.areSame( '😻 :smiling_cat_face_with_heart-eyes:', autocomplete.view.element.getChild( 0 ).getText(), 'First element in view should start from "smiling".' );
+				assert.areSame( '😻 smiling_cat_face_with_heart-eyes' , autocomplete.view.element.getChild( 0 ).getText(), 'First element in view should start from "smiling".' );
+			} );
+		},
+
+		// (#2583)
+		'test emoji autocomplete panel displays name': function( editor, bot ) {
+			emojiTools.runAfterInstanceReady( editor, bot, function( editor, bot ) {
+				bot.setHtmlWithSelection( '<p>foo 😻 :smiling_cat_face_with_heart-eyes:^</p>' );
```

Comment:
```suggestion
				bot.setHtmlWithSelection( '<p>foo :smiling_cat_face_with_heart-eyes:^</p>' );
```
The emoji in this place is a little bit confusing for me, as we only compare result in autocomplete panel.